tools/partitions/default_16MB.csv

```diff
@@ -3,4 +3,4 @@ nvs,      data, nvs,     0x9000,  0x5000,
 otadata,  data, ota,     0xe000,  0x2000,
 app0,     app,  ota_0,   0x10000, 0x640000,
 app1,     app,  ota_1,   0x650000,0x640000,
-spiffs,   data, spiffs,  0xc90000,0x370000,
+spiffs,   data, spiffs,  0xc90000,0x36FC00,
```

Comment:
@CoryCharlton - please test it with:

spiffs, data, spiffs, 0xc90000,0x36FFFC,